packages/itwinui-css/backstop/tests/table.html

```diff
@@ -124,6 +124,16 @@
         }
       );
     </script>
+    <script>
+      customElements.define(
+        'svg-filetype-video',
+        class extends HTMLElement {
+          connectedCallback() {
+            this.innerHTML = `<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><path d="m11 3v-3h-9v16h12v-13z" fill="#e64821"/><path d="m7 7.2919 3 1.7081-3 1.756zm5 1.7081a4 4 0 1 1 -4-4 4 4 0 0 1 4 4zm-1 0a3 3 0 1 0 -3 3 3 3 0 0 0 3-3z" fill="#fff"/><path d="m11 0 3 3h-3z" fill="#a13217"/></svg>`;
+          }
+        }
+      );
```

Comment:
I'm slightly new to this. Could you please explain what these lines of code do?

Reply:
Have you tried to google? It's creating a custom element called `<svg-filetype-video>` so that when the page loads, the svg code is placed inside it.

https://developer.mozilla.org/en-US/docs/Web/Web_Components
https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements